packages/react-jsx-highcharts/src/components/BaseChart/BaseChart.js

```diff
@@ -85,7 +85,7 @@ class BaseChart extends Component {
   render () {
     return (
       <div
-        className="chart"
+        className={`chart ${this.props.className}`}
```

Comment:
Thanks for the PR! 😄 

Mind adding a default empty string prop for `className`? Otherwise we'll end up having `class="chart undefined"`.

I'd do it myself but I only have my phone at the moment!